codetransformer/code.py

```diff
@@ -275,7 +275,7 @@ def from_pycode(cls, co):
             elif instr.uses_varname:
                 instr.arg = co.co_varnames[instr.arg]
             elif instr.uses_free:
-                instr.arg = _freevar_argname(
+                instr.arg, instr.vartype = _freevar_argname(
```

Comment:
Can we make a classmethod for these instrs that accepts this info also? I would like this to be supported for custom built code objects.